From 841ea64c9e35d41faee69b7bfd1b2fc3f3d23894 Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Thu, 5 Apr 2012 16:57:52 +0800
Subject: [PATCH 501/691] ENGR00178989: Disable secrity boot configs.

Security boot need to use fuse item. Thus it should not be enabled as
default.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 include/configs/mx6q_arm2.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/configs/mx6q_arm2.h b/include/configs/mx6q_arm2.h
index cd03bf7..b3e10f1 100644
--- a/include/configs/mx6q_arm2.h
+++ b/include/configs/mx6q_arm2.h
@@ -33,7 +33,9 @@
 #define CONFIG_FLASH_HEADER_OFFSET 0x400
 #define CONFIG_MX6_CLK32	   32768
 
-#define CONFIG_SECURE_BOOT /*For Secure Boot*/
+/*
+#define CONFIG_SECURE_BOOT
+*/
 
 #define CONFIG_SKIP_RELOCATE_UBOOT
 
-- 
1.7.7.4

